bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/wifihost-thing-type.xml

```diff
@@ -11,6 +11,7 @@
 
 		<label>Wifi Device</label>
 		<description>Provides Wifi device reachability</description>
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
I hesitated a little with WirelessAccessPoint

Reply:
Waht is the main purpose of this device? If that is to bridge between the wired and wireless network, it should be `WirelessAccessPoint`. Otherwise it is fine to leave it as is.

Reply:
From your description i think NetwerkAppliance is best, as it seems something similar to the network binding where one can add a thing (network device) and it shows connectivity information like ping, lastseen etc.

Reply:
Ok, so my choices were fine.